src/cli/src/install/mod.rs

```diff
@@ -19,10 +19,14 @@ fn fluvio_base_dir() -> Result<PathBuf, CliError> {
         home::home_dir().ok_or_else(|| IoError::new(ErrorKind::NotFound, "Homedir not found"))?;
     let path = home.join(".fluvio");
 
-    if path.exists() {
-        return Ok(path);
+    match path.exists() {
+        true => Ok(path),
+        false => {
+            // Create the base dir if it doesn't exist yet (#718)
+            std::fs::create_dir(&path)?;
+            Ok(path)
+        },
```

Comment:
This will work if `FLUVIO_DIR` is not set, but if it is set (above), then it will return early before creating the directory.